modules/tensor_mechanics/test/tests/anisotropic_patch/tests

```diff
@@ -4,5 +4,8 @@
     input = 'anisotropic_patch_test.i'
     exodiff = 'anisotropic_patch_test_out.e'
     scale_refine = 1
+    issues = '#4716 #7555'
+    design = 'source/Materials/ComputeLinearElasticStress.md source/Materials/ComputeSmallStrain.md syntax/Modules/TensorMechanics/Master/index.md'
+    requirement = 'The ComputeLinearElasticStress class shall compute and output the xx, yy, zz, xy, yz, and zx stress components on an irregular hex mesh compused of seven elements which form a unit cube with one internal element.'
```

Comment:
That comment in the .i file doesn't really say what this is testing.  I'd go with something like this:

The mechanics system shall be capable of accurately computing the elastic response of an anisotropic elastic material where 9 components of a symmetric elasticity tensor are specified on an irregular patch of elements with total small strain assumptions.